macosx/HBTitle.h

```diff
@@ -11,6 +11,21 @@ NS_ASSUME_NONNULL_BEGIN
 @class HBChapter;
 @class HBPreset;
 
+@interface HBMetadata : NSObject
+
+@property (nonatomic, readonly) NSString *name;
+@property (nonatomic, readonly) NSString *artist;
+@property (nonatomic, readonly) NSString *composer;
+@property (nonatomic, readonly) NSString *releaseDate;
```

Comment:
If this can return a nil value, you should annotate it with "nullable".

Reply:
- [ ] set all properties annotated with nullable